module/Olcs/assets/js/inline/forms/message-categories.js

```diff
@@ -0,0 +1,12 @@
+OLCS.ready(function() {
+  "use strict";
+
+  var form = "form[name=NewMessage]";
+
+  OLCS.cascadeInput({
+    source: form + " #category",
+    dest: form + " #subCategory",
+    url: "/list/task-sub-categories",
+    emptyLabel: "Please select"
+  });
+});
```

Comment:
This can be removed because we don't use it on the form any more (Subjects took the place of SubCategories)
However, a merge PR is doesn't feel like the place to do the cleanup, have noted it for later